src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -2909,6 +2909,51 @@ curl -X GET \
 
 </Section>
 
+<Section>
+
+<Text>
+
+### Get Product Limits for Plan
+
+Each pricing plan has different resource limits (number of groups, agents). When the user wants to downgrade, this method allows checking what resources exceeded the limits for a given plan.
+
+#### Specifics
+
+|                 |                                                                                       |
+| --------------- | ------------------------------------------------------------------------------------- |
+| HTTP Method     | **GET**                                                                               |
+| Method URL      | `https://api.livechatinc.com/v3.5/configuration/action/check_product_limits_for_plan` |
+| Required scopes | `subscription.lc:rw`                                                                  |
+
+
+#### Request
+
+| Query string | Required | Data type                                                                      | Notes |
+| ------------ | -------- | ------------------------------------------------------------------------------ | ----- |
+| `plan`       | yes      | `string`| Possible values: `starter`, `team`, `enterprise`, `enterpriseplus`   |       |
```

Comment:
The "Possible values" message should be in the "Notes" column

Reply:
Fixed here https://github.com/livechat/livechat-public-docs/pull/1099/commits/6f739b3582105badd29eb91960f91be2053d317f